docs/content/about/contribute.md

```diff
@@ -50,17 +50,25 @@ If you intend to change the public API, or make any non-trivial changes to the i
 
 If you’re only fixing a bug, it’s fine to submit a pull request right away but we still recommend to file an issue detailing what you’re fixing. This is helpful in case we don’t accept that specific fix but want to keep track of the issue.
 
-### Project Guideline
+### Conventional Commits
 
-Note, this section shall receive additional comments as we move forward.
+The Nikita Git repository follows the [Conventional Commits](https://www.conventionalcommits.org) specification that provides an easy set of rules for creating an explicit commit history.
 
-* Configs are listed by alphabetical order
-* Configs 1st line in the form of "* `name` (arg1, arg2)   "
-* Configs arg list the accept types separated by "|", types are bool, string, obj, int, float
-* Configs types can be surrounded by square braket to indicate an array, eg: "[int]"
-* Configs don't list global configs
-* Argument are listed in provided order
-* First two arguments must always be "err" and "status"
+Here's how a commit message looks like:
+
+```text
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer(s)]
+```
+
+The `<type>` message follows [the Angular convention](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit) and must be one from the list: `build`, `chore`, `ci`, `docs`, `style`, `refactor`, `perf`, `test`. The `[optional scope]` message is associated to the directory name of [the available packages](https://github.com/adaltas/node-nikita/tree/master/packages). A scope is optional and is contained within parentheses, e.g., `feat(engine): ability to parse arrays`.
+
+Don't be surprised why you are prompted with an error message when commiting to the Nikita repository not respecting Conventional Commits. It is [Husky](https://typicode.github.io/husky/) who plugs itself into the Git hook's configuration and enforces the validation rules using [commitlint](https://commitlint.js.org/) before committing.
```

Comment:
Commit messages are automatically validated. Internal, we use husky which plugin into Git by registering a hook to call commitlint to validate the format of the messages..